### PR TITLE
add instruction to call `pub get`

### PR DESCRIPTION
**Pull request by pq**
/cc @mit-mit @devoncarew

### CODE REVIEW

**Comment by pq** [comment]
Fancy!

@devoncarew, @mit-mit: any preference?

**Comment by mit-mit** [comment]
`pub add` command is nice!

**Comment by pq** [comment]
Updated to use `dart pub add`.

@mit-mit: take a look?

File: README.md [hunk 1]
```diff
@@ -39,11 +39,11 @@ For new apps created with `dart create`, the lints are enabled by default.
 
 For existing apps or packages, take these steps to enable these lints:
 
-1. Add a *dev* dependency on this package to your `pubspec.yaml` file:
+1. Add a *dev* dependency on this package to your `pubspec.yaml` file.  At the root
+of your package:
```
Comment (mit-mit) [inline README.md:43]: Maybe "In a terminal, located at the root of your package"?

Comment (pq) [reply README.md]: Done.

**Review by mit-mit — APPROVED** [review]

**Pull request merged**